services/app-api/src/resolvers/questionResponse/createRateQuestion.test.ts

```diff
@@ -0,0 +1,213 @@
+        const stateServer = await constructTestPostgresServer()
+        const cmsServer = await constructTestPostgresServer({
+            context: {
+                user: cmsUser,
+            },
+        })
+        const submittedContractAndRate =
+            await createAndSubmitTestContractWithRate(stateServer)
+        const rateID =
+            submittedContractAndRate.packageSubmissions[0].rateRevisions[0]
+                .rateID
+        await unlockTestContract(
+            cmsServer,
+            submittedContractAndRate.id,
+            'Test unlock reason'
+        )
+        const rateQuestionRes = must(
+            await createTestRateQuestion(cmsServer, rateID)
+        )
+        const rateQuestion = rateQuestionRes?.data?.createRateQuestion
+
+        expect(rateQuestion.question).toEqual(
+            expect.objectContaining({
+                id: expect.any(String),
+                rateID: rateID,
+                division: 'DMCO',
+                documents: [
+                    {
+                        name: 'Test Question',
+                        s3URL: 's3://bucketname/key/test1',
+                    },
+                ],
+                addedBy: cmsUser,
+            })
+        )
+
+        await submitTestContract(
+            stateServer,
+            submittedContractAndRate.id,
+            'Test resubmit reason'
+        )
+        const rateQuestionRes2 = must(
+            await createTestRateQuestion(cmsServer, rateID, {
+                documents: [
+                    {
+                        name: 'Test Question 2',
+                        s3URL: 's3://bucketname/key/test1',
+                    },
+                ],
+            })
+        )
+        const rateQuestion2 = rateQuestionRes2?.data?.createRateQuestion
+
+        expect(rateQuestion2.question).toEqual(
+            expect.objectContaining({
+                id: expect.any(String),
+                rateID: rateID,
+                division: 'DMCO',
+                documents: [
+                    {
+                        name: 'Test Question 2',
+                        s3URL: 's3://bucketname/key/test1',
+                    },
+                ],
+                addedBy: cmsUser,
+            })
+        )
+    })
+    it('returns an error if the rate is in DRAFT', async () => {
+        const stateServer = await constructTestPostgresServer()
+        const cmsServer = await constructTestPostgresServer({
+            context: {
+                user: cmsUser,
+            },
+        })
+        const submittedContractAndRate = must(
+            await createAndUpdateTestContractWithRate(stateServer)
+        )
+
+        const rateID = submittedContractAndRate.draftRates?.[0].id
+
+        if (!rateID) {
+            throw new Error(
+                'Unexpected error: Rate not found in test draft contract and rate'
+            )
+        }
+
+        const rateQuestionRes = must(
```

Comment:
There should be no need for must() here, our convention is for createTest* functions to throw on any error

Reply:
This is why I've generally invoked the gql.execute stuff in the tests for the given API, and used the createTest* stuff in other tests that need them. I'd be open to changing this convention and adopting must instead but I think that would need to be a broader change